test/BaselineOutput/SingleDebug/Rff/featurized.tsv

```diff
@@ -6,7 +6,7 @@
 #@   col=RffVectorFloat:R4:9-14
 #@ }
 15	8:Label
-5	1	1	1	2	1	3	1	0	-0.157029659	-0.555585265	0.490177631	-0.305056125	0.35670203	-0.453979075
-5	4	4	5	7	10	3	2	0	-0.375955045	0.43816793	-0.5670244	0.108704455	-0.271485656	-0.5095379
-3	1	1	1	2	2	3	1	0	-0.08380841	-0.571235061	0.4856296	-0.312245429	0.389987826	-0.4257262
-6	8	8	1	3	4	3	7	0	-0.2813567	0.504154444	-0.266616732	-0.512102365	-0.5723418	-0.07588247
+5	1	1	1	2	1	3	1	0	0.3418174	-0.465289354	0.247575819	-0.5215741	0.576975167	-0.02080728
```

Comment:
I think you'll also need to change this in the release @abgoswam. You could also delete it, and move it to common.

Reply:
Thanks. Done.

---
In reply to: [256215459](https://github.com/dotnet/machinelearning/pull/2524#discussion_r256215459) [](ancestors = 256215459)